app/models/car.rb

```diff
@@ -35,4 +35,21 @@ def slug_candidates
       [:company, :registered_number, :car_type]
     ]
   end
+
+  def currently_rent_by
+    if car_status == "rent"
+      rent = Rent.where(car_id: id).last.user
```

Comment:
you can avoid this 
Rent.where(car_id: id)
and directly use **rents** as there's direct association of car and rents, also here you're fetching rent records of a specific car. 
so try **rents** instead of whole where query and then last.user